conans/client/tools/apple.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import subprocess
+
+
+def cmd_output(command):
```

Comment:
it doesn't look like an apple thing

Reply:
where to put it?

Reply:
Not sure, maybe we just need to improve the `conanfile.run` method and not add another one.